sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -128,11 +182,63 @@ augmented AS (
     SUM(scalar_parent_browser_engagement_tab_open_event_count) OVER w1 AS tab_open_event_count_sum,
     SUM(active_ticks / (3600 / 5)) OVER w1 AS active_hours_sum,
     SUM(scalar_parent_browser_engagement_total_uri_count) OVER w1 AS total_uri_count,
-    COALESCE(client_experiments.experiments, []) AS experiments,
+    COALESCE(client_map_sums.experiments, []) AS experiments,
```

Comment:
Since these are all arrays they won't be null so coalesce shouldn't be needed along with the `EXCEPT`.  I think I was the one who put the coalesce on experiments but that was probably because my sql wasn't too good.